src/components/settings/withings-sync-status.tsx

```
@@ -0,0 +1,353 @@
+  useEffect(() => {
+    loadSyncStatus();
+    
+    // Auto-refresh every 30 seconds if there are active jobs
+    const interval = setInterval(() => {
+      if (syncData?.syncJobs.some(job => job.status === 'processing' || job.status === 'pending')) {
+        loadSyncStatus();
+      }
+    }, 30000);
+
+    return () => clearInterval(interval);
+  }, [syncData?.syncJobs, loadSyncStatus]);
```

Biome (2.1.2)

[error] This variable is used before its declaration.

The variable is declared here:


(lint/correctness/noInvalidUseBeforeDeclaration)